sources/org.osbuild.files

```diff
@@ -34,10 +34,12 @@ def fetch(url, checksum, directory):
     # subdirectory, so we avoid copying accross block devices.
     with tempfile.TemporaryDirectory(prefix="osbuild-unverified-file-", dir=directory) as tmpdir:
         # some mirrors are broken sometimes. retry manually, because curl doesn't on 404
-        for _ in range(3):
+        for _ in range(10):
             curl = subprocess.run([
                 "curl",
                 "--silent",
+                "--connect-timeout", "10",
+                "--max-time", "45",
```

Comment:
I'm not a huge fan of timeouts. It's very hard to find the right numbers. For example, 45 seconds might be too little for some connections when downloading the largest packages (`linux-firmware` in Fedora 31 is ~240 MB). What are we guarding against - a too slow download?

`--connect-timeout` makes sense to quickly weed out mirrors that don't respond at all.

Reply:
Ugh, good point. I think we only need the `--connect-timeout` one. Fixed it.